crates/pink/sidevm/host-runtime/src/env.rs

```diff
@@ -483,6 +495,11 @@ impl env::OcallFuncs for State {
             Query => create_channel!(self.query_tx),
         }
     }
+
+    fn gas_remaining(&mut self) -> Result<u8> {
```

Comment:
Should we just return all the detailed info to the guest (i.e. `self.gas_to_breath` and `self.init_gas_to_breath`)?

Reply:
Returning a type can be cast into an i32 is the fastest way.
We can consider another detailed API in the future if it is needed.

Reply:
OK